tests/test_connector.rs

```diff
@@ -12,6 +12,11 @@ use byte_slice_cast::AsByteSlice;
 use near_sdk_sim::transaction::ExecutionStatus;
 use primitive_types::U256;
 
+pub type PausedMask = u8;
+const UNPAUSE_ALL: PausedMask = 0;
+const PAUSE_DEPOSIT: PausedMask = 1 << 0;
```

Comment:
I still don't like all this code duplication between production and tests. Nothing to do about it in this PR, but we'll have to refactor this in the future.

Reply:
I agree!

Reply:
Just adding to technical debt eh :D?